scripts/exgfs_atmos_goes_nawips.sh

```diff
@@ -50,56 +46,50 @@ pdsext=no
 
 sleep_interval=20
 max_tries=180
-fhr=${fstart}
-for (( fhr=fstart; fhr <= fend; fhr=fhr+finc )); do
-  fhr3=$(printf "%03d" "${fhr}")
-  GRIBIN="${COM_ATMOS_GOES}/${model}.${cycle}.${GRIB}${fhr3}${EXT}"
-  GEMGRD="${RUN2}_${PDY}${cyc}f${fhr3}"
-
-  GRIBIN_chk="${GRIBIN}"
-
-  if ! wait_for_file "${GRIBIN_chk}" "${sleep_interval}" "${max_tries}"; then
-    echo "FATAL ERROR: after 1 hour of waiting for ${GRIBIN_chk} file at F${fhr3} to end."
-    export err=7 ; err_chk
-    exit "${err}"
-  fi
-
-  cp "${GRIBIN}" "grib${fhr3}"
-
-  export pgm="nagrib_nc F${fhr3}"
-
-   ${NAGRIB} << EOF
-   GBFILE   = grib${fhr3}
-   INDXFL   = 
-   GDOUTF   = ${GEMGRD}
-   PROJ     = ${proj}
-   GRDAREA  = ${grdarea}
-   KXKY     = ${kxky}
-   MAXGRD   = ${maxgrd}
-   CPYFIL   = ${cpyfil}
-   GAREA    = ${garea}
-   OUTPUT   = ${output}
-   GBTBLS   = ${gbtbls}
-   GBDIAG   = 
-   PDSEXT   = ${pdsext}
-  l
-  r
-EOF
-  export err=$?;err_chk
 
-  "${GEMEXE}/gpend"
-
-  cpfs "${GEMGRD}" "${COM_ATMOS_GEMPAK_0p25}/${GEMGRD}"
-  if [[ ${SENDDBN} == "YES" ]] ; then
-      "${DBNROOT}/bin/dbn_alert" MODEL "${DBN_ALERT_TYPE}" "${job}" \
-			     "${COM_ATMOS_GEMPAK_0p25}/${GEMGRD}"
-  else
-      echo "##### DBN_ALERT_TYPE is: ${DBN_ALERT_TYPE} #####"
-  fi
+fhr3=$(printf "%03d" "${fhr}")
+
+GEMGRD="${RUN2}_${PDY}${cyc}f${fhr3}"
+GRIBIN="${COM_ATMOS_GOES}/${model}.${cycle}.${GRIB}${fhr3}${EXT}"
+GRIBIN_chk="${GRIBIN}"
+
+if ! wait_for_file "${GRIBIN_chk}" "${sleep_interval}" "${max_tries}"; then
+  echo "FATAL ERROR: after 1 hour of waiting for ${GRIBIN_chk} file at F${fhr3} to end."
+  export err=7 ; err_chk
+  exit "${err}"
+fi
```

Comment:
Now that we're doing one forecast hour at a time, the job isn't triggered until this file should exist, so I think we can just check it once and fail if it isn't there.
```suggestion
if [[ ! -r "${GRIBIN_chk}" ]]; then
  echo "FATAL ERROR: GRIB index file ${GRIBIN_chk} not found!"
  export err=7 ; err_chk
  exit "${err}
fi
```